pkg/plugins/golang/v4/scaffolds/internal/templates/main.go

```diff
@@ -221,11 +221,14 @@ func init() {
 }
 
 func main() {
-	var metricsAddr string
-	var enableLeaderElection bool
-	var probeAddr string
-	var secureMetrics bool
-	var enableHTTP2 bool
+	var (
+		metricsAddr          string
+		enableLeaderElection bool
+		probeAddr            string
+		secureMetrics        bool
+		enableHTTP2          bool
+		tlsOpts              []func(*tls.Config)
+	)
```

Comment:
```suggestion
	var metricsAddr string
	var enableLeaderElection bool
	var probeAddr string
	var secureMetrics bool
	var enableHTTP2 bool
	var tlsOpts []func(*tls.Config)
```

That ^ @fengshunli and then just run make generate

Reply:
I don't understand why the var code block is not used here to define it. Isn't the code block definition more standardized? @camilamacedo86

Reply:
What value do add with this change?
It will only make harder for people upgrade/update their project with the latest changes 
It has no need to change it in the scaffold. Lets try to only do in each PR what is required to achieve the goal of the PR itself.